src/__tests__/index.js

```diff
@@ -106,7 +106,7 @@ test(
   'should reduce additions and subtractions (7)',
   testValue(
     'calc(0px - (24px - (var(--a) - var(--b)) / 2 + var(--c)))',
-    'calc(-24px + var(--a)/2 - var(--b)/2 - var(--c))'
```

Comment:
Can't do this for if `--a` or `--b` resolves to an addition or subtraction. The same is true for all other examples.